auto_nag/utils.py

```diff
@@ -74,6 +74,20 @@ def get_config(name, entry, default=None):
     return tool_conf.get(entry, default)
 
 
+def get_receivers(tool_name):
+    receiver_lists = get_config("common", "receiver_list", default={})
+
+    receivers = get_config(tool_name, "receivers", [])
+    if isinstance(receivers, six.string_types):
```

Comment:
Nit: we no longer need to use six given that we only support Python 3 now, we can just go with `isinstance(receivers, str)`

Reply:
Done in e757368caa42905ca19890dc4f87f2cf118dfc8c